src/lib/test-helpers.ts

```diff
@@ -0,0 +1,3 @@
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+export const noop = () => {}
```

Comment:
noop and identity will likely be a commonly used thing if we expand the tests and we don't have a need to spy on the actual calls